File: 1.Sum-of-Digits-using-char.c
int main()
{
    int t=0,i=0,ans=0,count=0; 
    char a0 = '0';
    char n[7];
    scanf("%d",&t);
    while(t--){
        ans=0;
        scanf("%s",n);
        count = strlen(n);
        for(i=0;i<count;i++){
            ans+= (n[i] - a0);
        }
        printf("%d\n",ans);
    }
    return 0;
}

File: 1.Sum-of-Digits-using-char.go
package main 
import (
"fmt"
"strings"
"strconv"
)

func main(){
    var t int;
    var ans int;
    var n string;
    fmt.Scanf("%d",&t);
    for(t>0) {
        fmt.Scanf("%s",&n);
        var list = strings.Split(n,"");
        var i=0;
        ans = 0;
        for(i<len(list)){
            w, err := strconv.Atoi(list[i]);
            j, err := strconv.Atoi("0");
            ans += w + j;
            i++;
            //fmt.Println(err);
            _ = err;
        }
        fmt.Println(ans)
        t = t - 1;
     }
}
